src/client/Microsoft.Identity.Client/AuthScheme/PoP/PopAuthenticationOperation.cs

```diff
@@ -86,6 +88,14 @@ public void FormatResult(AuthenticationResult authenticationResult)
             authenticationResult.AccessToken = popToken;
         }
 
+        public Task FormatResultAsync(AuthenticationResult authenticationResult, CancellationToken cancellationToken = default)
```

Comment:
for now should we also return a `Task.FromCanceled(cancellationToken)` if one was passed?